src/status_im/contexts/communities/events.cljs

```diff
@@ -478,3 +508,28 @@
 (rf/reg-event-fx :communities/update-last-opened-at-success
  (fn [{:keys [db]} [community-id last-opened-at]]
    {:db (assoc-in db [:communities community-id :last-opened-at] last-opened-at)}))
+
+(rf/defn share-community
+  {:events [::share-community-confirmation-pressed]}
```

Comment:
Since you're touching this code, prefer not to use namespaced keywords to stay consistent with the guidelines.

Reply:
Done